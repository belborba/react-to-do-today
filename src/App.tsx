import { MainContainer } from "./components/MainContainer/MainContainer";
import { useTheme } from "./contexts/ThemeContext";

function App() {
  const themeCtx = useTheme();

  return (
    <div
      className={`w-screen h-screen ${
        themeCtx?.theme === "dark"
          ? "bg-zinc-800 text-zinc-50"
          : "bg-zinc-400 text-zinc-800"
      } p-4 flex items-center justify-center`}
    >
      <MainContainer />
    </div>
  );
}

export default App;
