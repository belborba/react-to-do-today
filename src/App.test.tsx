import { render } from "@testing-library/react";
import App from "./App";

jest.mock("./pages/Main", () => ({
  Main: () => <div>Main</div>,
}));

describe("App", () => {
  it("renders without crashing", () => {
    render(<App />);
    expect(document.body).toContainHTML("Main");
  });
});
