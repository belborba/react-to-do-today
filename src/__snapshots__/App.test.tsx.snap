// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`App renders without crashing 1`] = `
<body>
  <div>
    <div
      class="flex h-screen w-screen items-center justify-center bg-zinc-300 p-4 text-zinc-50 dark:bg-zinc-900 dark:text-zinc-800"
    >
      <div
        class="font-quicksand sm:4/5 lx:w-4/5 box-border flex h-full w-full flex-col gap-6 rounded-2xl border border-zinc-300 bg-white/50 px-4 py-6 sm:p-10 md:w-4/5 md:p-10 lg:w-4/5 dark:border-zinc-500 dark:bg-zinc-800/80"
      >
        <div
          class="flex w-full flex-col items-center space-y-4 border-b border-zinc-300 pb-4 sm:flex-row sm:items-start sm:justify-between sm:space-y-0 dark:border-zinc-500"
        >
          <svg
            class="fill-zinc-800 dark:fill-violet-400"
            height="39"
            viewBox="0 0 238 39"
            width="238"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M235.382 8.83008C236.082 8.83012 236.663 9.07352 237.121 9.55859C237.579 10.0166 237.808 10.5965 237.808 11.2969V28.4775C237.808 30.9569 237.336 32.9246 236.393 34.3799C235.476 35.862 234.237 36.9264 232.674 37.5732C231.111 38.22 229.359 38.5439 227.419 38.5439C226.557 38.5439 225.639 38.4765 224.669 38.3418C223.699 38.207 222.904 38.0048 222.284 37.7354C221.476 37.385 220.909 36.9403 220.586 36.4014C220.29 35.8895 220.236 35.3368 220.425 34.7441C220.667 33.9628 221.058 33.4234 221.597 33.127C222.136 32.8575 222.702 32.8305 223.295 33.0459C223.726 33.1807 224.293 33.3829 224.993 33.6523C225.694 33.9487 226.503 34.0967 227.419 34.0967C228.658 34.0966 229.682 33.9215 230.49 33.5713C231.326 33.2479 231.945 32.6816 232.35 31.873C232.781 31.0915 232.997 30.014 232.997 28.6396V28.209C232.726 28.5452 232.43 28.8518 232.107 29.125C231.434 29.6908 230.626 30.1225 229.683 30.4189C228.739 30.7154 227.66 30.8633 226.447 30.8633C225.019 30.8632 223.766 30.5401 222.688 29.8936C221.637 29.2198 220.816 28.3024 220.223 27.1436C219.63 25.9578 219.333 24.6097 219.333 23.1006V11.2969C219.333 10.5964 219.563 10.0167 220.021 9.55859C220.479 9.07364 221.058 8.83014 221.759 8.83008C222.459 8.83017 223.039 9.07357 223.497 9.55859C223.955 10.0167 224.184 10.5964 224.184 11.2969V21.6455C224.184 23.3967 224.561 24.6365 225.315 25.3643C226.097 26.065 227.176 26.416 228.551 26.416C229.494 26.4159 230.289 26.2408 230.936 25.8906C231.582 25.5134 232.081 24.9739 232.432 24.2734C232.782 23.5459 232.956 22.6694 232.956 21.6455V11.2969C232.956 10.5964 233.186 10.0166 233.644 9.55859C234.102 9.07364 234.681 8.83017 235.382 8.83008Z"
            />
            <path
              clip-rule="evenodd"
              d="M31.8486 8.42676C33.8968 8.42676 35.7438 8.91165 37.3877 9.88184C39.0585 10.825 40.3784 12.146 41.3486 13.8438C42.3457 15.5146 42.8447 17.4553 42.8447 19.665C42.8447 21.8748 42.3457 23.8286 41.3486 25.5264C40.3784 27.1973 39.0586 28.505 37.3877 29.4482C35.7438 30.3914 33.8968 30.8633 31.8486 30.8633C29.8009 30.8631 27.9411 30.3913 26.2705 29.4482C24.6268 28.505 23.3057 27.1971 22.3086 25.5264C21.3386 23.8287 20.8536 21.8746 20.8535 19.665C20.8536 17.4554 21.3386 15.5145 22.3086 13.8438C23.3057 12.146 24.6268 10.8251 26.2705 9.88184C27.9411 8.91181 29.8008 8.4269 31.8486 8.42676ZM31.8486 12.873C30.7172 12.8732 29.6803 13.1569 28.7373 13.7227C27.8211 14.2885 27.0796 15.0835 26.5137 16.1074C25.9748 17.1045 25.7041 18.2908 25.7041 19.665C25.7042 21.0124 25.9747 22.1986 26.5137 23.2227C27.0796 24.2197 27.8211 25.0005 28.7373 25.5664C29.6803 26.1322 30.7171 26.4159 31.8486 26.416C32.9805 26.416 34.0056 26.1324 34.9219 25.5664C35.8649 25.0005 36.6066 24.2197 37.1455 23.2227C37.7114 22.1986 37.9941 21.0124 37.9941 19.665C37.9941 18.2908 37.7113 17.1044 37.1455 16.1074C36.6066 15.0835 35.8649 14.2886 34.9219 13.7227C34.0056 13.1567 32.9805 12.873 31.8486 12.873Z"
              fill-rule="evenodd"
            />
            <path
              clip-rule="evenodd"
              d="M76.1279 0.543945C76.8283 0.544065 77.4082 0.772527 77.8662 1.23047C78.3241 1.68859 78.5537 2.28227 78.5537 3.00977V27.9922C78.5537 28.6928 78.3242 29.2864 77.8662 29.7715C77.4082 30.2295 76.8284 30.4589 76.1279 30.459C75.4272 30.459 74.8468 30.2296 74.3887 29.7715C73.9423 29.2987 73.7154 28.7234 73.7041 28.0459C73.3729 28.4052 72.9683 28.7657 72.4893 29.125C71.8427 29.6099 71.0745 30.0277 70.1855 30.3779C69.3233 30.7013 68.3794 30.8632 67.3555 30.8633C65.4961 30.8633 63.8108 30.3913 62.3018 29.4482C60.7928 28.478 59.5934 27.1571 58.7041 25.4863C57.8417 23.7885 57.4102 21.8479 57.4102 19.665C57.4102 17.4553 57.8418 15.5146 58.7041 13.8438C59.5934 12.146 60.7796 10.8251 62.2617 9.88184C63.7438 8.91182 65.388 8.42682 67.1934 8.42676C68.3519 8.4268 69.4167 8.60192 70.3867 8.95215C71.3567 9.30246 72.1929 9.74723 72.8936 10.2861C73.1929 10.5079 73.4621 10.7327 73.7021 10.959V3.00977C73.7021 2.30926 73.9308 1.72958 74.3887 1.27148C74.8468 0.786372 75.4272 0.543945 76.1279 0.543945ZM67.9619 12.873C66.803 12.873 65.778 13.1698 64.8887 13.7627C63.9994 14.3556 63.2991 15.1645 62.7871 16.1885C62.3021 17.2125 62.0596 18.3716 62.0596 19.665C62.0596 20.9314 62.3022 22.0767 62.7871 23.1006C63.2991 24.1245 63.9995 24.9335 64.8887 25.5264C65.778 26.1193 66.803 26.416 67.9619 26.416C69.1473 26.4159 70.1841 26.119 71.0732 25.5264C71.9626 24.9335 72.6516 24.1247 73.1367 23.1006C73.6485 22.0768 73.9042 20.9312 73.9043 19.665C73.9043 18.3717 73.6486 17.2124 73.1367 16.1885C72.6516 15.1643 71.9626 14.3556 71.0732 13.7627C70.1842 13.1702 69.1472 12.8731 67.9619 12.873Z"
              fill-rule="evenodd"
            />
            <path
              clip-rule="evenodd"
              d="M93.791 8.42676C95.8388 8.42685 97.6844 8.91187 99.3281 9.88184C100.999 10.8251 102.321 12.1459 103.291 13.8438C104.288 15.5145 104.786 17.4554 104.786 19.665C104.786 21.8746 104.288 23.8287 103.291 25.5264C102.321 27.1973 100.999 28.505 99.3281 29.4482C97.6844 30.3912 95.8388 30.8632 93.791 30.8633C91.7429 30.8633 89.8828 30.3914 88.2119 29.4482C86.568 28.505 85.2472 27.1973 84.25 25.5264C83.28 23.8286 82.795 21.8747 82.7949 19.665C82.795 17.4553 83.2799 15.5146 84.25 13.8438C85.2471 12.1459 86.568 10.8251 88.2119 9.88184C89.8828 8.9117 91.7429 8.42676 93.791 8.42676ZM93.791 12.873C92.6591 12.873 91.621 13.1567 90.6777 13.7227C89.7616 14.2885 89.02 15.0835 88.4541 16.1074C87.9153 17.1044 87.6465 18.2909 87.6465 19.665C87.6465 21.0124 87.9152 22.1987 88.4541 23.2227C89.02 24.2196 89.7616 25.0005 90.6777 25.5664C91.621 26.1324 92.6591 26.416 93.791 26.416C94.9227 26.4159 95.9462 26.1322 96.8623 25.5664C97.8054 25.0005 98.547 24.2197 99.0859 23.2227C99.6518 22.1986 99.9355 21.0124 99.9355 19.665C99.9355 18.2908 99.6518 17.1045 99.0859 16.1074C98.547 15.0835 97.8054 14.2885 96.8623 13.7227C95.9462 13.1569 94.9226 12.8731 93.791 12.873Z"
              fill-rule="evenodd"
            />
            <path
              clip-rule="evenodd"
              d="M186.271 0.543945C186.971 0.543949 187.552 0.77233 188.01 1.23047C188.468 1.6886 188.696 2.28223 188.696 3.00977V27.9922C188.696 28.6928 188.468 29.2864 188.01 29.7715C187.552 30.2296 186.971 30.459 186.271 30.459C185.57 30.4589 184.99 30.2296 184.532 29.7715C184.086 29.299 183.858 28.7239 183.847 28.0469C183.516 28.4059 183.112 28.766 182.633 29.125C181.986 29.6101 181.217 30.0276 180.328 30.3779C179.466 30.7013 178.522 30.8633 177.498 30.8633C175.639 30.8632 173.954 30.3913 172.445 29.4482C170.936 28.478 169.737 27.1571 168.848 25.4863C167.985 23.7885 167.554 21.8479 167.554 19.665C167.554 17.4554 167.985 15.5145 168.848 13.8438C169.737 12.146 170.923 10.8251 172.405 9.88184C173.887 8.91187 175.531 8.42692 177.336 8.42676C178.495 8.42676 179.561 8.60179 180.531 8.95215C181.501 9.30245 182.337 9.74728 183.037 10.2861C183.336 10.5077 183.605 10.7328 183.845 10.959V3.00977C183.845 2.30923 184.074 1.72959 184.532 1.27148C184.99 0.786454 185.57 0.544027 186.271 0.543945ZM178.105 12.873C176.947 12.873 175.922 13.1698 175.032 13.7627C174.143 14.3556 173.443 15.1644 172.931 16.1885C172.446 17.2125 172.203 18.3716 172.203 19.665C172.203 20.9314 172.446 22.0767 172.931 23.1006C173.443 24.1246 174.143 24.9335 175.032 25.5264C175.922 26.1193 176.947 26.416 178.105 26.416C179.291 26.4159 180.329 26.1192 181.218 25.5264C182.107 24.9335 182.794 24.1246 183.279 23.1006C183.791 22.0767 184.047 20.9314 184.047 19.665C184.047 18.3716 183.791 17.2125 183.279 16.1885C182.794 15.1644 182.107 14.3556 181.218 13.7627C180.329 13.1699 179.291 12.8731 178.105 12.873Z"
              fill-rule="evenodd"
            />
            <path
              clip-rule="evenodd"
              d="M211.655 8.42676C212.356 8.4268 212.935 8.65614 213.394 9.11426C213.851 9.57219 214.081 10.1645 214.081 10.8916V27.9922C214.081 28.6929 213.852 29.2864 213.394 29.7715C212.935 30.2296 212.356 30.4589 211.655 30.459C210.955 30.459 210.374 30.2296 209.916 29.7715C209.47 29.299 209.243 28.7238 209.231 28.0469C208.9 28.406 208.495 28.7659 208.017 29.125C207.37 29.6099 206.602 30.0277 205.713 30.3779C204.851 30.7013 203.907 30.8633 202.883 30.8633C201.024 30.8632 199.339 30.3913 197.83 29.4482C196.321 28.478 195.121 27.1573 194.231 25.4863C193.369 23.7886 192.938 21.8478 192.938 19.665C192.938 17.4554 193.369 15.5145 194.231 13.8438C195.121 12.146 196.307 10.8251 197.789 9.88184C199.271 8.91169 200.916 8.42676 202.722 8.42676C203.88 8.42681 204.945 8.60184 205.915 8.95215C206.885 9.30243 207.72 9.7473 208.421 10.2861C208.72 10.5078 208.989 10.7327 209.229 10.959V10.8916C209.23 10.1915 209.458 9.61219 209.916 9.1543C210.374 8.6692 210.955 8.42678 211.655 8.42676ZM203.489 12.873C202.331 12.873 201.306 13.1699 200.417 13.7627C199.528 14.3556 198.826 15.1644 198.314 16.1885C197.829 17.2125 197.587 18.3716 197.587 19.665C197.587 20.9313 197.83 22.0767 198.314 23.1006C198.826 24.1246 199.528 24.9335 200.417 25.5264C201.306 26.1192 202.331 26.416 203.489 26.416C204.675 26.416 205.713 26.1193 206.603 25.5264C207.492 24.9335 208.179 24.1245 208.664 23.1006C209.176 22.0767 209.432 20.9313 209.432 19.665C209.432 18.3716 209.176 17.2125 208.664 16.1885C208.179 15.1645 207.492 14.3556 206.603 13.7627C205.713 13.1698 204.675 12.873 203.489 12.873Z"
              fill-rule="evenodd"
            />
            <path
              clip-rule="evenodd"
              d="M152.44 8.3252C154.489 8.32524 156.335 8.81107 157.979 9.78125C159.649 10.7245 160.97 12.0445 161.94 13.7422C162.938 15.4131 163.436 17.3545 163.436 19.5645C163.435 21.7741 162.937 23.728 161.94 25.4258C160.97 27.0966 159.649 28.4034 157.979 29.3467C156.335 30.2899 154.489 30.7617 152.44 30.7617C150.392 30.7617 148.532 30.29 146.861 29.3467C145.218 28.4035 143.897 27.0965 142.9 25.4258C141.93 23.728 141.444 21.7742 141.444 19.5645C141.444 17.3545 141.93 15.4131 142.9 13.7422C143.897 12.0447 145.218 10.7244 146.861 9.78125C148.532 8.81102 150.392 8.3252 152.44 8.3252ZM157.498 21.1328C157.132 20.894 156.642 20.9968 156.402 21.3623C155.731 22.3872 154.566 23.1422 152.801 23.2324L152.44 23.2412C150.471 23.2412 149.193 22.4555 148.478 21.3623C148.238 20.9967 147.748 20.8938 147.382 21.1328C147.016 21.3722 146.914 21.8628 147.153 22.2285C148.194 23.8187 150 24.8242 152.44 24.8242L152.891 24.8135C155.104 24.6972 156.752 23.7193 157.728 22.2285C157.966 21.8628 157.864 21.3722 157.498 21.1328ZM150.329 15.8525C149.892 15.8525 149.537 16.2065 149.537 16.6436V18.7549C149.537 19.1921 149.892 19.5469 150.329 19.5469C150.766 19.5468 151.121 19.1921 151.121 18.7549V16.6436C151.121 16.2066 150.766 15.8526 150.329 15.8525ZM154.551 15.8525C154.114 15.8528 153.76 16.2067 153.76 16.6436V18.7549C153.76 19.192 154.114 19.5466 154.551 19.5469C154.988 19.5468 155.343 19.1921 155.343 18.7549V16.6436C155.343 16.2066 154.988 15.8526 154.551 15.8525Z"
              fill-rule="evenodd"
            />
            <path
              d="M20.415 2.16016C21.1155 2.16025 21.6953 2.3765 22.1533 2.80762C22.6382 3.23874 22.8808 3.80443 22.8809 4.50488C22.8809 5.20553 22.6383 5.77193 22.1533 6.20312C21.6953 6.60725 21.1155 6.8085 20.415 6.80859H14.0674V27.9922C14.0674 28.6929 13.8118 29.2864 13.2998 29.7715C12.8147 30.2295 12.1809 30.459 11.3994 30.459C10.6451 30.4589 10.025 30.2295 9.54004 29.7715C9.05506 29.2864 8.8125 28.6928 8.8125 27.9922V6.80859H2.46582C1.7654 6.80854 1.17248 6.59403 0.6875 6.16309C0.229391 5.73192 4.56307e-05 5.16547 0 4.46484C2.11233e-05 3.76424 0.229462 3.21185 0.6875 2.80762C1.17253 2.37652 1.76527 2.16021 2.46582 2.16016H20.415Z"
            />
            <path
              d="M139.362 2.16016C140.063 2.16024 140.643 2.37649 141.101 2.80762C141.585 3.23873 141.828 3.80448 141.828 4.50488C141.828 5.20551 141.586 5.77193 141.101 6.20312C140.643 6.60731 140.063 6.80851 139.362 6.80859H133.016V27.9922C133.016 28.6928 132.759 29.2864 132.247 29.7715C131.762 30.2295 131.128 30.459 130.347 30.459C129.592 30.4589 128.972 30.2295 128.487 29.7715C128.002 29.2864 127.76 28.6928 127.76 27.9922V6.80859H121.412C120.712 6.80841 120.12 6.59382 119.635 6.16309C119.177 5.73192 118.947 5.16547 118.947 4.46484C118.947 3.76417 119.177 3.21186 119.635 2.80762C120.12 2.37664 120.712 2.16034 121.412 2.16016H139.362Z"
            />
          </svg>
          <div
            class="flex items-center gap-3"
          >
            <button
              class="relative flex h-8 w-8 cursor-pointer items-center justify-center p-1.5 bg-zinc-800 dark:bg-violet-400 rounded-full"
            >
              <svg
                class="text-zinc-300 dark:text-zinc-800"
                data-name="Camada 1"
                id="Camada_1"
                viewBox="0 0 178.09 178.09"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M122.1,83.67c16.03-.39,31.56.31,43.01-13.04,7.61-8.89,8.1-22.42.62-31.58-1.19-1.46-2.93-2.73-4.18-4.18-2.2-1.35-4.31-2.83-6.76-3.74-2.56-.95-4.93-.95-7.57-1.28.07-3.77-.81-7.28-2.63-10.58-.7-1.27-1.62-2.3-2.22-3.56l-4.18-4.18c-9.37-7.71-22.47-7.2-31.58.62-12.98,11.14-12.5,25.71-13.03,41.29-.34,10.05-.07,20.17-.23,30.23,9.57-.18,19.18.25,28.75.02Z"
                  fill="currentColor"
                />
                <path
                  d="M55.99,92.74c-16.03.39-31.56-.31-43.01,13.04-7.61,8.89-8.1,22.42-.62,31.58,1.19,1.46,2.93,2.73,4.18,4.18,2.2,1.35,4.31,2.83,6.76,3.74,2.56.95,4.93.95,7.57,1.28-.07,3.77.81,7.28,2.63,10.58.7,1.27,1.62,2.3,2.22,3.56l4.18,4.18c9.37,7.71,22.47,7.2,31.58-.62,12.99-11.15,12.49-25.71,13.03-41.29.35-10.05.08-20.17.23-30.23-9.57.18-19.18-.25-28.75-.02Z"
                  fill="currentColor"
                />
                <path
                  d="M54.27,83.66c10.05.35,20.17.08,30.23.23-.18-9.57.25-19.18.02-28.75-.39-16.03.31-31.56-13.04-43.01-8.89-7.61-22.42-8.1-31.58-.62-1.46,1.19-2.73,2.93-4.18,4.18-1.35,2.2-2.83,4.31-3.74,6.76-.95,2.56-.95,4.93-1.28,7.57-3.77-.07-7.28.81-10.58,2.63-1.27.7-2.3,1.62-3.56,2.22l-4.18,4.18c-7.71,9.37-7.2,22.47.62,31.58,11.16,13,25.68,12.49,41.29,13.03Z"
                  fill="currentColor"
                />
                <path
                  d="M165.11,105.78c-11.15-12.99-25.71-12.49-41.29-13.03-10.05-.35-20.17-.08-30.23-.23.18,9.57-.25,19.18-.02,28.75.39,16.03-.31,31.56,13.04,43.01,8.89,7.61,22.42,8.1,31.58.62,1.46-1.19,2.73-2.93,4.18-4.18,1.35-2.2,2.83-4.31,3.74-6.76.95-2.56.95-4.93,1.28-7.57,3.77.07,7.28-.81,10.58-2.63,1.27-.7,2.3-1.62,3.56-2.22l4.18-4.18c7.71-9.37,7.2-22.47-.62-31.58Z"
                  fill="currentColor"
                />
              </svg>
            </button>
            <p
              class="bg-zinc-300 text-zinc-800 dark:bg-zinc-700 dark:text-zinc-50 p-2 px-4 rounded-full"
            >
              July 22, 2025
            </p>
            <button
              class="relative flex h-[40px] w-[80px] cursor-pointer flex-col items-end gap-4 rounded-full p-1 transition duration-200 ease-in-out items-start bg-zinc-300"
            >
              <span
                class="absolute flex h-8 w-8 items-center justify-center rounded-full bg-zinc-700 p-1.5 transition duration-500 ease-in-out dark:bg-violet-400"
              >
                <svg
                  fill="none"
                  height="18"
                  viewBox="0 0 17 18"
                  width="17"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8.3125 13.7002C8.62133 13.7042 8.87074 13.9539 8.875 14.2627V16.5127C8.875 16.8234 8.62316 17.0752 8.3125 17.0752C8.00184 17.0752 7.75 16.8234 7.75 16.5127V14.2627C7.75426 13.9539 8.00367 13.7042 8.3125 13.7002ZM4.20215 12.3281C4.42371 12.1217 4.76917 12.1278 4.9834 12.3418C5.19761 12.556 5.20359 12.9014 4.99707 13.123L3.40723 14.7129C3.30232 14.8191 3.15906 14.8786 3.00977 14.8779C2.86038 14.8794 2.71667 14.8198 2.6123 14.7129C2.39318 14.4933 2.39306 14.1375 2.6123 13.918L4.20215 12.3281ZM11.6416 12.3418C11.8558 12.1278 12.2013 12.1217 12.4229 12.3281L14.0127 13.918C14.2319 14.1375 14.2318 14.4933 14.0127 14.7129C13.9083 14.8198 13.7646 14.8794 13.6152 14.8779C13.4659 14.8786 13.3227 14.8191 13.2178 14.7129L11.6279 13.123C11.4214 12.9014 11.4274 12.556 11.6416 12.3418ZM8.3125 5.2627C10.3834 5.2627 12.0623 6.94182 12.0625 9.0127C12.0625 11.0838 10.3836 12.7627 8.3125 12.7627C6.24143 12.7627 4.5625 11.0838 4.5625 9.0127C4.56273 6.94182 6.24157 5.2627 8.3125 5.2627ZM3.0625 8.4502C3.37133 8.45423 3.62074 8.70392 3.625 9.0127C3.62096 9.32166 3.37147 9.57115 3.0625 9.5752H0.8125C0.50184 9.5752 0.25 9.32336 0.25 9.0127C0.250231 8.70223 0.501982 8.4502 0.8125 8.4502H3.0625ZM15.8125 8.4502C16.123 8.4502 16.3748 8.70223 16.375 9.0127C16.375 9.32336 16.1232 9.5752 15.8125 9.5752H13.5625C13.2518 9.5752 13 9.32336 13 9.0127C13.0002 8.70223 13.252 8.4502 13.5625 8.4502H15.8125ZM13.2178 3.31348C13.358 3.16299 13.5693 3.10047 13.7686 3.15137C13.9677 3.20234 14.123 3.35841 14.1738 3.55762C14.2247 3.75687 14.1631 3.96821 14.0127 4.1084L12.46 5.69824C12.3551 5.80442 12.2118 5.86399 12.0625 5.86328C11.9005 5.87463 11.7417 5.81411 11.6279 5.69824C11.4086 5.47863 11.4086 5.12293 11.6279 4.90332L13.2178 3.31348ZM2.85645 3.15137C3.05575 3.10047 3.267 3.16299 3.40723 3.31348L4.99707 4.90332C5.21641 5.12293 5.21641 5.47863 4.99707 5.69824C4.77744 5.91731 4.42167 5.91749 4.20215 5.69824L2.6123 4.1084C2.46185 3.96821 2.40034 3.75687 2.45117 3.55762C2.50205 3.35841 2.65727 3.20234 2.85645 3.15137ZM8.3125 0.950195C8.62302 0.950195 8.87477 1.20223 8.875 1.5127V3.7627C8.87096 4.07166 8.62147 4.32115 8.3125 4.3252C8.00353 4.32115 7.75404 4.07166 7.75 3.7627V1.5127C7.75023 1.20223 8.00198 0.950195 8.3125 0.950195Z"
                    fill="white"
                  />
                </svg>
              </span>
            </button>
          </div>
        </div>
        <div
          class="relative w-full"
        >
          <input
            class="w-full rounded-2xl bg-zinc-50 p-5 pr-12 text-zinc-800 outline-none focus:ring-2 sm:text-2xl dark:bg-zinc-700 dark:text-zinc-50 dark:focus:ring-violet-300"
            placeholder="What are your plans for today?"
            type="text"
            value=""
          />
          <button
            class="font-semiboldmd:text-2xl absolute top-1/2 right-4 flex h-[30px] w-[30px] -translate-y-1/2 cursor-pointer items-center justify-center rounded-md bg-zinc-800 p-2 sm:h-[40px] sm:w-[40px] sm:text-2xl md:h-[40px] md:w-[40px] dark:bg-violet-400 dark:text-zinc-800"
          >
            +
          </button>
        </div>
        <div
          class="scrollbar-hide flex h-full w-full touch-manipulation flex-col items-center overflow-y-scroll rounded-2xl"
        >
          <ul
            class="flex w-full flex-col gap-4"
          >
            <li
              aria-describedby="DndDescribedBy-1"
              aria-disabled="false"
              aria-roledescription="sortable"
              class="flex w-full items-center justify-between gap-4 rounded-2xl bg-zinc-50/70 p-4 hover:bg-zinc-50 dark:bg-zinc-700 dark:hover:bg-zinc-700"
              role="button"
              style="opacity: 1; cursor: grab;"
              tabindex="0"
            >
              <div
                class="flex items-center"
              >
                <input
                  class="h-[24px] w-[24px] rounded-md accent-zinc-800 dark:accent-violet-400"
                  type="checkbox"
                />
                <p
                  class="inline rounded-2xl pl-4 text-zinc-800 dark:text-zinc-50"
                >
                  Example: Buy a cake
                </p>
              </div>
              <svg
                fill="none"
                height="20"
                viewBox="0 0 17 20"
                width="17"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M16.0001 3.33333H13.4167C13.2233 2.39284 12.7116 1.54779 11.9678 0.940598C11.224 0.333408 10.2936 0.0012121 9.33341 0H7.66675C6.70658 0.0012121 5.77618 0.333408 5.03237 0.940598C4.28856 1.54779 3.77683 2.39284 3.58341 3.33333H1.00008C0.779068 3.33333 0.567106 3.42113 0.410826 3.57741C0.254545 3.73369 0.166748 3.94565 0.166748 4.16667C0.166748 4.38768 0.254545 4.59964 0.410826 4.75592C0.567106 4.9122 0.779068 5 1.00008 5H1.83341V15.8333C1.83474 16.938 2.27415 17.997 3.05526 18.7782C3.83638 19.5593 4.89542 19.9987 6.00008 20H11.0001C12.1047 19.9987 13.1638 19.5593 13.9449 18.7782C14.726 17.997 15.1654 16.938 15.1667 15.8333V5H16.0001C16.2211 5 16.4331 4.9122 16.5893 4.75592C16.7456 4.59964 16.8334 4.38768 16.8334 4.16667C16.8334 3.94565 16.7456 3.73369 16.5893 3.57741C16.4331 3.42113 16.2211 3.33333 16.0001 3.33333ZM7.66675 1.66667H9.33341C9.85031 1.6673 10.3544 1.82781 10.7764 2.1262C11.1985 2.42459 11.5179 2.84624 11.6909 3.33333H5.30925C5.48223 2.84624 5.80167 2.42459 6.22374 2.1262C6.64581 1.82781 7.14985 1.6673 7.66675 1.66667ZM13.5001 15.8333C13.5001 16.4964 13.2367 17.1323 12.7678 17.6011C12.299 18.0699 11.6631 18.3333 11.0001 18.3333H6.00008C5.33704 18.3333 4.70116 18.0699 4.23231 17.6011C3.76347 17.1323 3.50008 16.4964 3.50008 15.8333V5H13.5001V15.8333Z"
                  fill="#71717B"
                />
                <path
                  d="M6.83348 14.9995C7.0545 14.9995 7.26646 14.9117 7.42274 14.7554C7.57902 14.5991 7.66681 14.3872 7.66681 14.1661V9.16615C7.66681 8.94514 7.57902 8.73318 7.42274 8.5769C7.26646 8.42062 7.0545 8.33282 6.83348 8.33282C6.61247 8.33282 6.40051 8.42062 6.24423 8.5769C6.08795 8.73318 6.00015 8.94514 6.00015 9.16615V14.1661C6.00015 14.3872 6.08795 14.5991 6.24423 14.7554C6.40051 14.9117 6.61247 14.9995 6.83348 14.9995Z"
                  fill="#71717B"
                />
                <path
                  d="M10.1667 14.9995C10.3877 14.9995 10.5997 14.9117 10.756 14.7554C10.9122 14.5991 11 14.3872 11 14.1661V9.16615C11 8.94514 10.9122 8.73318 10.756 8.5769C10.5997 8.42062 10.3877 8.33282 10.1667 8.33282C9.94569 8.33282 9.73372 8.42062 9.57744 8.5769C9.42116 8.73318 9.33336 8.94514 9.33336 9.16615V14.1661C9.33336 14.3872 9.42116 14.5991 9.57744 14.7554C9.73372 14.9117 9.94569 14.9995 10.1667 14.9995Z"
                  fill="#71717B"
                />
              </svg>
            </li>
          </ul>
          <div
            id="DndDescribedBy-1"
            style="display: none;"
          >
            
    To pick up a draggable item, press the space bar.
    While dragging, use the arrow keys to move the item.
    Press space again to drop the item in its new position, or press escape to cancel.
  
          </div>
          <div
            aria-atomic="true"
            aria-live="assertive"
            id="DndLiveRegion-0"
            role="status"
            style="position: fixed; top: 0px; left: 0px; width: 1px; height: 1px; margin: -1px; border: 0px; padding: 0px; overflow: hidden; clip-path: inset(100%); white-space: nowrap;"
          />
        </div>
        <div
          id="DndDescribedBy-0"
          style="display: none;"
        >
          
    To pick up a draggable item, press the space bar.
    While dragging, use the arrow keys to move the item.
    Press space again to drop the item in its new position, or press escape to cancel.
  
        </div>
        <div
          aria-atomic="true"
          aria-live="assertive"
          id="DndLiveRegion-1"
          role="status"
          style="position: fixed; top: 0px; left: 0px; width: 1px; height: 1px; margin: -1px; border: 0px; padding: 0px; overflow: hidden; clip-path: inset(100%); white-space: nowrap;"
        />
      </div>
    </div>
  </div>
</body>
`;
